e2e/cypress/tests/integration/archived_channel/archive_channel_search_spec.ts

```diff
@@ -45,7 +45,7 @@ describe('archive tests while preventing viewing archived channels', () => {
         const messageText = `pineapple ${getRandomId()}`;
 
         // # Archive the channel
-        createArchivedChannel({prefix: 'pineapple-'}, [messageText]).then(() => {
+        createArchivedChannel({prefix: 'pineapple-'}, [messageText], []).then(() => {
```

Comment:
Please convert the helper file where `createArchivedChannel` can be found and define types for its parameters with `messages` and `memberUsernames` as optional.

Reply:
Same, fixed in b1cbaae6cf563bbeb3070efb22f97a05c8787855